libs/types/src/tabulation.ts

```diff
@@ -42,7 +42,7 @@ export type CandidateContestResults = ContestResultsBase & {
 export type ContestResults = YesNoContestResults | CandidateContestResults;
 
 /**
- * Should match `CVR.vxBallotStyle`.
+ * Should match `CVR.vxBallotStyle` from the generated CDF cast vote record types.
```

Comment:
Ah, the test you added made me realise that this should be `vxBallotType` instead of `vxBallotStyle` (which is why I didn't find it in my search earlier)

Anyway, FWIW, looks like we can create the union type from the enum, in that was the blocker:
```ts
type VotingMethod = `${vxBallotType}`;
```
(https://stackoverflow.com/a/52396706)

Reply:
nice! updated it